test/e2e/suite_test.go

```diff
@@ -237,6 +237,13 @@ func afterTest(t *testing.T, namespace string, f *framework.E2eFramework, deploy
 }
 
 func cleanUp(t *testing.T, namespace string, f *framework.E2eFramework, deployTraefik bool) error {
+
+	if t.Failed() {
```

Comment:
We don't need to dump the logs here since it is done in f.DumpClusterInfo()`. The functions are nice have though in case we want to dump logs at point before the test finishes. It would be nice to have them written to a directory like `DumpClusterInfo` but this is a secondary concern right now.